app/templates/profile.html

```diff
@@ -258,15 +258,44 @@ <h6 class="card-title text-center mb-1">{{ user.full_name }}</h6>
                 <!-- End Features card -->
 
               </div>
-
-              <!-- Center -->
-              <div class="col-5">
-                <div class="container" id="daily_horoscope">
+                            <!-- Center -->
+                            <div class="col-5">
+                                <div class="container" id="daily_horoscope">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
                 </div>
-              </div>
             </div>
```

Comment:
Why the reformatting?

Reply:
This file was a bit "broken".
I tried to fix it. 
But now there is new pr that ode opened :)